docs/pages/machine-id/getting-started.mdx

```diff
@@ -0,0 +1,120 @@
+
+Download the appropriate version of Teleport for your platform from
+our [downloads page](https://goteleport.com/teleport/download).
+
+You will be using the `tbot` binary to from the package you download and install.
+
+## Step 2/4. Create Bot User
+
+Before you create a bot user, you need to determine which role you want to assign to the bot user. You can use `tctl` to examine what roles exist on your system.
+
+```bash
+$ tctl get roles --format=text
+Role    Allowed to login as                           Node Labels Access to resources                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           
+------- --------------------------------------------- ----------- -----------------------------------------
+access  {{internal.logins}}                           <all nodes> event:list,read,session:read,list                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             
+auditor no-login-6566121f-b602-47f1-a118-c9c618ee5aec             session:list,read,event:list,read                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             
+editor                                                            user:list,create,read,update,delete,...
+```
+
+The above output is from a fresh install of Teleport with the default roles, your cluster may have additional roles. In our example, let’s assume you want to give the bot `access` role to allow it to connect to SSH servers within your cluster.
```

Comment:
Are we allowing the bot to connect to SSH servers, or we allowing the bot to issue certificates that allow other machines to connect to SSH servers within the cluster?